src/components/header/header.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {cleanup, render} from '@testing-library/react';
+import Header from './header';
+
+afterEach(cleanup);
+
+it('Header', () => {
```

Comment:
tests should have this structure
```suggestion
describe('Header', () => {
    it('should render', () => { test code });
    
    it('should redirect on the button click', () => { test code })
});
```